.github/workflows/run-tests.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04, windows-2019]
         php: [8.1, 8.0, 7.4, 7.3]
-        laravel: [8.*]
+        laravel: [8.*, 9.*]
```

Comment:
Same as above

Reply:
yes i see because laravel 9 is only php8+ compatible but i'm not that much familiar with github actions to split php matrix... can you help?

Reply:
You've to add an `exclude` key, see https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix for an example

Reply:
thx! ok i've updated the PR